php-sync/sync.php

```diff
@@ -318,26 +318,25 @@ public function scheduleBatches ($mode, $toSchedule) {
 
 			$nodecfg = $this->config["NODES"][$node];
 			$bulkMaxBatches = !empty($nodecfg["BULK_MAX_BATCHES"]) ? $nodecfg["BULK_MAX_BATCHES"] : $this->config["BULK_MAX_BATCHES"];
-			
+
 			$batches = array();
 			for ($nBatch=0; $nBatch < $bulkMaxBatches*2; $nBatch++) {
 				$batchName = readdir($hBatchDir);
 				if ($batchName === FALSE) {
 					break;
 				}
-				
+
 				$batches[] = $batchName;
 			}
-			
+
 			closedir($hBatchDir);
-			
+
 
 			$rowno = 0;
-			$bulk = array ();
+			$match = $bulk = array();
 			$bulkCount = 0;
 			$lastType = null;
 			foreach ($batches as $batch) {
-				unset($match);
```

Comment:
Unsets like this are intentional because there is a loop. I'm interested in the variable being clear outside of the loop. Same goes for every other unset.